### ISSUE TITLE
Adding a new project to the Backlog

### ISSUE THREAD

**Issue by ncsokas** [ncsokas/github-actions-training#113]
Create a new card in the Backlog list with proper descriptions of the project

Fields to populate:
Description about the tool
Similar known artifacts:
Reason for assessing (What problem is it solving?):
Research tasks: (Examples: How is it maintained, price, OS?, where does it fit in the ecosystem, how will it be used?)
Prerequisite: What should be present before the spike
Starting litterature
